walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,12 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    name = name.trim();
```

Comment:
would recommend adding more docs to the method to describe in a little more detail what it does. Also, what is the expectation when the name is empty or `null`? (if it can be either)

Reply:
Sounds good! And I don't think the method accounts for that! I'll add some code to account for that